Sources/Sentry/SentryUIViewControllerSwizzling.m

```diff
@@ -95,19 +106,6 @@ - (void)start
                                  @"a rootViewController");
             }
         }
-
-        [self swizzleAllSubViewControllersInApp:app];
-    } else {
-        // If we can't find an UIApplication instance we may use the current process path as the
-        // image name. This mostly happens with SwiftUI projects.
-        NSString *processImage = self.processInfoWrapper.processPath;
```

Comment:
`h`: Is it save to remove this fallback?

Reply:
IMO it is. the app binary itself will be in the `inAppInclude` automatically.

Reply:
I thought so, too. Just wanted to double-check.